parity/rpc_apis.rs

```diff
@@ -566,6 +570,13 @@ impl ApiSet {
 				public_list.insert(Api::Personal);
 				public_list
 			},
+			ApiSet::PubSub => vec![
```

Comment:
You cannot consume elements from an array or slice afaict. `into_iter()` is implemented only for `&` and returns `IntoIterator<Item=&T>`, so we would need to clone the items.